exampleconfigs/core-fabric-example.yaml

```diff
@@ -0,0 +1,328 @@
+ # available loggers: TBD
+ spec: debug
+
+# ------------------- FSC Node Configuration -------------------------
+fsc:
+  # The FSC id provides a name for this node instance within the fsc network and is used when
+  # naming docker resources for nwo testing as well as the unique id for tracing.
+  # It's also used as the unique name to resolve this node's identity and grpc server endpoint
+  id: someid
+
+  # This is the identity of the node
+  identity:
+    cert:
+      file: /path/to/cert.pem
+    key:
+      file: /path/to/key.pwm
+  
+  # TBD
+  client:
+    certs:
+    - path/to/client1-cert.pem
+
+  # TBD
+  admin:
+    certs:
+    - path/to/admin-cert.pem
+
+  # ------------------- GRPC Server Configuration -------------------------
+  grpc:
+    # The listen address of this server
+    address: 0.0.0.0:20000
+    # ConnectionTimeout specifies the timeout for connection establishment for all new connections
+    connectionTimeout: 10s
+
+    tls:
+      # Whether TLS is enabled or not
+      enabled: true
+      # Whether clients are required to provide their TLS certificates for verification
+      clientAuthRequired: false
+      # TLS Certificate
+      cert:
+        file: /path/to/tls/server.crt 
+      # TLS Key
+      key:
+        file: /path/to/tls/server.key
+      
+      # Root certificates to be able to verify client TLS certificates, only required
+      # if clientAuthRequired is set to true
+      clientRootCAs:
+        files:
+        - /path/to/client/tls/ca.crt
+
+    # GRPC Server keepalive parameters
+    keepalive:
+      # minInterval is the minimum permitted time between client pings.
+      # If clients send pings more frequently, the peer server will
+      # disconnect them
+      minInterval: 60s
+      # interval is the duration after which if the server does not see
+      # any activity from the client it pings the client to see if it's alive
+      interval: 300s
+      # Timeout is the duration the server waits for a response
+      # from the client after sending a ping before closing the connection
+      timeout: 600s
+
+  # ------------------- P2P Configuration -------------------------
+  p2p:
+    # listen address for see https://github.com/libp2p/specs/blob/master/addressing/README.md
+    # for information on the format of the listenAddress
+    listenAddress: /dns4/borrower/tcp/20001
+    # bootstrap node
+    # if it's empty then this node is the bootstrap node, otherwise it's the name
+    # of the bootstrap node, which must be defined in the FSC endpoint resolvers section
+    # and that entry must have an address with an entry P2P.
+    bootstrapNode: approver
+
+  # ------------------- KVS Configuration -------------------------
+  # Internal key/value store used by the node to store information
+  # such as bindings (eg resolvers)
+  kvs:
+    cache:
+      # Sets the maximum number of cached items
+      size:
```

Comment:
empty size, will it be set to 0 in that case?

Reply:
That's a good point, if any of the entries doesn't have a value, what is it's default. I will look at adding those